File: Java/src/AddBinaryTest.java
import org.junit.Assert;
import org.junit.Test;

public class AddBinaryTest {

    @Test
    public void testAddBinary() {
        AddBinary ab = new AddBinary();

        Assert.assertEquals(ab.addBinary("0", "0"), "0");

        Assert.assertEquals(ab.addBinary("11", "1"), "100");

        Assert.assertEquals(ab.addBinary("101", "101"), "1010");
    }

}

File: Kotlin/src/AddBinaryTest.kt
import org.junit.Test

class AddBinaryTest {

    @Test
    fun testAddBinary() {
        val ab = AddBinary()

        assert(ab.addBinary("0", "0") == "0")

        assert(ab.addBinary("11", "1") == "100")

        assert(ab.addBinary("101", "101") == "1010")
    }

}